src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/SdkClient/ResourceManagerSdkClient.cs

```diff
@@ -607,6 +608,28 @@ public virtual List<PSResourceGroup> FilterResourceGroups(string name, Hashtable
                     resourceGroups.AddRange(listResult);
                 }
 
+                if (!string.IsNullOrEmpty(name))
```

Comment:
ToDo:  This would be really nice extracted out into a helper like:

```C#
IEnumerable<T> FilterByWildCard<T>(IEnumerable<T> source, Func<T, string> getKey, string parameterValue)
```